### PR TITLE
Improve setup instructions for Windows

### PR DESCRIPTION
**Pull request by lakrsv**
# Introduction
This pull request enhances the setup instructions for installing OS dependencies for Windows, located at https://bevyengine.org/learn/book/getting-started/setup/#install-os-dependencies

This fixes the issue I raised here: https://github.com/bevyengine/bevy-website/issues/622

## Before
 
before_bevy

## After
after_bevy

# Environment
**Operating System**: Windows 10 Pro 64-Bit (10.0, Build 19045)
**Processor**: AMD Ryzen 7 1800X Eight-Core Processor

### CODE REVIEW

File: content/learn/book/getting-started/setup/_index.md [hunk 1]
```diff
@@ -24,9 +24,24 @@ Once this is done, you should have the ```rustc``` compiler and the ```cargo```
...
+
+#### Windows
+
+* Run the [Visual Studio 2019 build tools installer](https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=BuildTools&rel=16)
+* For easy setup, select the ```Desktop development with C++``` workload in the installer.
+* For a mimimal setup, follow these steps:
+    1. In the installer, navigate to ```Individual components```
+    2. Select the latest ```MSVC``` for your architecture and version of Windows
+    3. Select the latest ```Windows SDK``` for your version of Windows
+    4. Select the ```C++ CMake tools``` for Windows component
```
Comment (james7132) [inline content/learn/book/getting-started/setup/_index.md:39]:
```suggestion
* For easy setup, select the ```Desktop development with C++``` workload in the installer.
* For a mimimal setup, follow these steps:
    1. In the installer, navigate to `Individual components`
    2. Select the latest `MSVC` for your architecture and version of Windows
    3. Select the latest `Windows SDK` for your version of Windows
    4. Select the `C++ CMake tools` for Windows component
```

Comment (lakrsv) [reply content/learn/book/getting-started/setup/_index.md]: Thanks, I've applied that @james7132

**Review by james7132 — APPROVED** [review]

**Comment by IceSentry** [comment]
I feel like this was merged a bit too quickly. I believe this should have linked to this article instead https://rust-lang.github.io/rustup/installation/windows-msvc.html

Also, the visual studio link should probably point to 2022 not 2019

**Comment by IceSentry** [comment]
If we prefer keeping the info in there, it should at least mention using rustup-init to avoid all the manual steps.

**Comment by cart** [comment]
> I feel like this was merged a bit too quickly. I believe this should have linked to this article instead https://rust-lang.github.io/rustup/installation/windows-msvc.html

I agree with this. The less we are re-documenting Rust environment setup, the better.

**Pull request merged**